### PR TITLE
feat: new payment flow explanation

### PR DESCRIPTION
**Pull request by marco-kuama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,41 +7,18 @@ In its current implementation the integration is composed of two different imple
...
-
-```
 ## REST API Integration
 
 ### Authentication
-In order to initialize a new payment session, it is necessary to perform a very basic integration to Toonie's Authentication endpoint.
 
+In order to initialize a new payment session, it is necessary to perform a very basic integration to an Authentication endpoint.
+
+Then you need to call the endpoint to create the payment session by passing some parameters, like an amount, a currency and a reason.
```
Comment (gmcabiato) [inline README.md]: In the Authentication paragraph? I'd move this out

File: README.md [hunk 2]
```diff
@@ -7,41 +7,18 @@ In its current implementation the integration is composed of two different imple
...
+You need to pass it a success and an error url too, appending to it the `?orderId={CHECKOUT_ORDER_ID}` which will be replaced in our system with the right value.
```
Comment (gmcabiato) [inline README.md]:
```suggestion
You need to pass it a success and an error url too. You can use the `{PAYMENT_SESSION_ID}` placeholder anywhere in your url, which will be replaced in our system with the right value.
```

File: README.md [hunk 3]
```diff
@@ -58,318 +35,57 @@ const tokenRes = await fetch('https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/
...
-      "amount": "1.1",
-      "walletId": "<MERCHANTWALLETID>",
-      "currency": "EUR",
-      "paymentSessionId": paymentSessionId,
-      "reason": "Test Payment 01"
+      amount: amount ?? "1.23",
+      currency: currency ?? "EUR",
+      reason: reason ?? "Test Payment 01",
+      "successUrl": "https://www.success.com?orderId={CHECKOUT_ORDER_ID}",
+      "errorUrl": "https://www.error.com?orderId={CHECKOUT_ORDER_ID}",
```
Comment (gmcabiato) [inline README.md]:
```suggestion
      "errorUrl": "https://www.yourdomain.com/error/?orderId={PAYMENT_SESSION_ID}",
```

File: README.md [hunk 4]
```diff
@@ -58,318 +35,57 @@ const tokenRes = await fetch('https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/
...
+      Authorization: ` Bearer ${tokenData.access_token}`,
       "content-type": "application/json",
     },
     body: JSON.stringify({
-      "amount": "1.1",
-      "walletId": "<MERCHANTWALLETID>",
-      "currency": "EUR",
-      "paymentSessionId": paymentSessionId,
-      "reason": "Test Payment 01"
+      amount: amount ?? "1.23",
```
Comment (gmcabiato) [inline README.md:58]: For this and all other values
```suggestion
      "amount": amount ?? "1.23",
```

File: README.md [hunk 5]
```diff
@@ -58,318 +35,57 @@ const tokenRes = await fetch('https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/
...
-const createCardPaymentIntent = async (paymentSessionId) => {
-  const tokenData = await getTokenData();
-
-  const res = await fetch('https://<ENVIRONMENT_API_URL>/acquiring/v1/card/custom', {
-    method: 'POST',
+  const res = await fetch("https://<ENVIRONMENT_API_URL>/acquiring/v1/payment", {
+    method: "POST",
     headers: {
-      Authorization: `Bearer ${tokenData.access_token}`,
+      Authorization: ` Bearer ${tokenData.access_token}`,
```
Comment (gmcabiato) [inline README.md]:
```suggestion
      "Authorization": `Bearer ${tokenData.access_token}`,
```

Comment (kuamanet) [reply README.md]: No need to add the `"`, it's js not json :)

**Pull request merged**